com.tibia.Tibia.metainfo.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.tibia.Tibia</id>
+
+  <name>Tibia</name>
+  <summary>Become a part of Tibia, the MMORPG classic that has been expanded for more than 25 years</summary>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+
+  <recommends>
+    <display_length compare="ge">1020</display_length>
+  </recommends>
+
+  <description>
+     <p>
+      Tibia is one of the oldest and most successful massively multiplayer online role-playing games (MMORPG) created in Europe. In an MMORPG people from all over the world meet on a virtual playground to explore areas, solve tricky riddles and undertake heroic exploits.
+    </p>
+    <p>
+      For more than 25 years now, players have been visiting the medieval world of Tibia. At present, more than 500,000 players from all over the world inhabit the Tibian continent enjoying the numerous game features.
+    </p>
+    <p>
+      Acting as knights, paladins, sorcerers or druids, players are faced with the challenge of developing the skills of their selected characters, exploring a large variety of areas and dangerous dungeons and interacting with other players on a social and diplomatic level. Besides the sophisticated chat tools it is especially the unique freedom players enjoy in Tibia that create an enormously immersive gaming experience.
+    </p>
+    <p>
+      NOTE: This wrapper is not verified by, affiliated with, or supported by Cipsoft GmbH.
+    </p>
```

Comment:
Put it as the first para.

Reply:
OK, but the generator is using it in a different order.